From 94bd10e1bb5b76cf54a35ee1740379dc85255e35 Mon Sep 17 00:00:00 2001
From: Rohit Athavale <rathaval@xilinx.com>
Date: Fri, 23 Feb 2018 11:15:07 -0800
Subject: [PATCH 0306/1852] Documentation: media: Correct dt-bindings doc for
 Gamma LUT IP

commit 5249a21c15a1ed12e5b2fde88ae63dbdeb3347b2 from
https://github.com/Xilinx/linux-xlnx.git

Correct the IP name from scaler to Gamma LUT and update for
8 and 10-bit color depths.

Signed-off-by: Rohit Athavale <rathaval@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/media/xilinx/xlnx,v-gamma-lut.txt     | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Documentation/devicetree/bindings/media/xilinx/xlnx,v-gamma-lut.txt b/Documentation/devicetree/bindings/media/xilinx/xlnx,v-gamma-lut.txt
index 67229fa81a16..f45fe6102c59 100644
--- a/Documentation/devicetree/bindings/media/xilinx/xlnx,v-gamma-lut.txt
+++ b/Documentation/devicetree/bindings/media/xilinx/xlnx,v-gamma-lut.txt
@@ -15,7 +15,7 @@ Required properties:
 - reset-gpios: Specifier for a GPIO that asserts Gamma IP (AP_RST_N) reset
 
 - ports: Video ports, using the DT bindings defined in ../video-interfaces.txt.
-  The scaler has an input port (0) and an output port (1).
+  The Gamma LUT IP has an input port (0) and an output port (1).
 
 
 Required port properties:
@@ -23,7 +23,7 @@ Required port properties:
   A Sink Pad is represented by reg = <0>
   A Source Pad is represented by reg = <1>
 
-- xlnx,video-width: Video width as defined in video.txt
+- xlnx,video-width: Video width as defined in video.txt. Can be either 8 or 10.
 
 Example :
 
-- 
2.31.1

